W 12430553 logging combine third party gr (#196)
mulesoft/docs-gateway: gateway/1.4/modules/ROOT/pages/flex-local-monitoring.adoc

= Monitoring in Local Mode
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images

> Local Mode enables you enables you to monitor Flex Gateway APIs independent of Anypoint Platform. For Flex Gateways running in local mode, you can configure different Fluent Bit logs.

Reviewer: ```suggestion
Flex Gateway running in Local Mode enables you to monitor APIs independent of Anypoint Platform. For Flex Gateway running in Local Mode, you can deliver logs to different Fluent Bit outputs.
```

Reviewer 2: I agree with Chris' suggestions.

> Get started monitoring Flex Gateway running in Local Mode with the following tasks:

Reviewer 2: See previous similar comment.

> * xref:flex-local-third-party-logs-config.adoc[Configure Logs for Flex Gateway in Local Mode] - Flex Gateway enables you to configure log output for Fluent Bit output types. Logs outputs types, such as `File`, `HTTP`, `Splunk`. Logs for Third-Party servicesservices are configure via a configuration YAML file.

Reviewer 2: ```suggestion
* xref:flex-local-third-party-logs-config.adoc[Configure Logs for Flex Gateway in Local Mode] - Flex Gateway enables you to configure log output for Fluent Bit output types, such as `File`, `HTTP`, and `Splunk`. Use a configuration YAML file to configure logs for third-party services.
```

> * xref:flex-local-view-logs[View Logs] - Flex Gateway maintains standard output logs regardless of logging configuration. Use the standard output logs for quick logging checks where an additional logging output is not needed, such as ensuring an external service is running or API requests are reaching Flex Gateway.

Reviewer 2: ```suggestion
* xref:flex-local-view-logs[View Logs] - Flex Gateway maintains standard output logs regardless of logging configuration. Use the standard output logs for quick logging checks when an additional logging output is not needed, such as when ensuring an external service is running or API requests are reaching Flex Gateway.
```

Reviewer 2: Make sure the bullets are parallel.
